ca_driver/src/create_driver.cpp

```diff
@@ -49,6 +49,12 @@ CreateDriver::CreateDriver(ros::NodeHandle& nh) : nh_(nh), priv_nh_("~")
   tf_odom_.child_frame_id = str_base_footprint;
   odom_msg_.header.frame_id = "odom";
   odom_msg_.child_frame_id = str_base_footprint;
+  joint_state_msg_.name.resize(2);
```

Comment:
should we populate `header.frame_id` too? I'm not familiar with *joint_state_publisher*